source/features/batch-open-conversations.tsx

```diff
@@ -63,12 +78,44 @@ async function init(): Promise<void | false> {
 				Open selected
 			</button>
 		);
+	} else {
+		select('#js-issues-toolbar')?.prepend(
+			<div className="mr-3 d-none d-md-block">
+  			<input type="checkbox" data-check-all="" aria-label="Select all issues" autoComplete="off"/>
+		  </div>
+		);
+		select('#js-issues-toolbar')?.append(
+			<div className="table-list-triage flex-auto js-issues-toolbar-triage">
+				<span className="text-gray table-list-header-toggle">
+					<span data-check-all-count="">1</span> selected
+					<button type="button" className="btn-link rgh-batch-open-issues pl-3">Open selected</button>
+				</span>
+			</div>
+		);
+
+		const prRows = select.all('.js-issue-row');
+		const prs = prRows.map(getIssueConfig);
+
+		for (const pr of prs) {
```

Comment:
You can inline getIssueConfig in here, no need for 2 separate loops

Reply:
i was adopting the code like this `getPRConfig`:

https://github.com/sindresorhus/refined-github/blob/daf633137ec08e6f5647a6f8bb0b02d5dce58ef9/source/features/conflict-marker.tsx#L50


but less code is always better to maintain 👍 
i do inline it

Reply:
That one has to loop because it’s preparing the data for the API call. Here it’s just 2 consecutive loops. Thanks for merging them

Reply:
ok - makes sense 👍 
i will inline the function code as well